### ISSUE TITLE
Fill in gaps / spell things out more in Collision Detection lesson

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1100]
tl;dr - I think a bit more teacher hand-holding in this lesson would be useful! This is complex stuff.

In doing the Instruction Box audit in this lesson, I'm noticing a few holes / things I would want to see in this lesson if I were teaching it. 

Namely, in the first section ("Problem Decomposition Returns!") we need to explain to teachers which strategy is top-down and which strategy is bottom-up. I do think it should go after the first synthesize question ("Which strategy did students use?") as an indented and italicized bullet.  Spelling out pros/ cons of each approach would be useful. (Alas, I am not the most qualified person to do this.)

I also think we should explain the second Synthesize question "Did they start out with one, and then switch to another." Some notes on why / when students would switch strategies would be helfpul.

I also think the wording on the retractable flag worksheet leads students to solve in one way rather than the other. Is that what we want? Height is listed first, so that's likely what students will define first! The lesson states that teachers should have students decide what strategy to use, but it is not printed on the worksheet that kids can try area then height, or height than area.

And finally, more teacher notes explicitly drawing the connections between the two sections of the lesson would be very useful. I can easily imagine a teacher having kids do the first worksheet, and then having kids do the second worksheet, and failing to help kids see how they are alike / different. Can we spell out the connections?

**Comment by schanzer** [comment]
@retabak ready for your review

**Comment by retabak** [comment]
I think the additions are very helpful. I think, however, that I would want even MORE guidance if I were going to teach this lesson for first time. 

When we ask, "Which strategy will YOU use - top-down, or bottom-up?", I think it would be nice to have some sample responses from students. I am struggling to anticipate what the sample responses would be, or I'd write them myself. I'd like to see an indented, italicized bullet beneath this question that says, "_Sample response: I will use top-down because.... I will use bottom up because...._" What are misconceptions kids have around this? It all feels a little nebulous still. (I haven't taught this lesson. It was too complex for my sixth graders.)

In "Synthesize," it's not obvious what you're getting at with this question: "Did they start out with one, and then switch to another?" Can you explain why / when that might happen? (I'm assuming it's a thing that kids do, otherwise we wouldn't inquire about it here.)

**Comment by schanzer** [comment]
@retabak I don't think adding student answers makes sense here, especially because the "why" isn't part of the question. I added more detail to that section though, so let me know if it addresses the concern?

In "Synthesize", I'm asking if a student tried going top-down, and then decided to switch to bottom-up. The latest commit attempts to clarify that as well. As for _why_ it might happen -- who knows?🤷‍♂️ We still ask people if they switched from top-down to bottom-up when they're drawing circles, and sometimes they have interesting answers.

If that question feels too open-ended, I can delete it. But I've always found it valuable during PD, for example.

**Comment by retabak** [comment]
I played around a little bit. This is starting to feel better!

One more thing that I would want as a teacher:
This question - "Connect this back to profit, revenue, cost and onscreen from Problem Decomposition lesson" - also feels like a big lift for students and teachers.  I think spelling out the big connections that you're hoping students will be able to make would be useful. Or we could provide some prompts for teachers to get students thinking about the connections? This is something I would want.  (Also, onscreen is not in the Problem Decomp lesson - is that a typo?)

**Comment by schanzer** [comment]
@retabak see latest push

**Comment by retabak** [comment]
Yes, this is what I needed.
Closing!
